server/views/pages/associatedStaffSelect.njk

```diff
@@ -0,0 +1,104 @@
+{% extends "../partials/layout.njk" %}
+{% from "../partials/breadCrumb.njk" import breadcrumb %}
+
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% set title = "Select a staff member" %}
+
+{% block beforeContent %}
+  {{ breadcrumb() }}
+{% endblock %}
+
+{% block content %}
+  {% if errors | length %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors,
+      attributes: { "data-qa": "error-summary" }
+    }) }}
+  {% endif %}
+
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+
+  <form method="POST" novalidate="novalidate" data-qa="search-form">
+  <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+
+  <div class="horizontal-form govuk-!-margin-bottom-3">
+    {{ govukInput({
```

Comment:
This has to match the inputs in the `assault` form doesn't it? 

If there are 2 inputs there - first and last name - then there needs to be 2 inputs here